apps/web/src/components/mention/actions.ts

```diff
@@ -0,0 +1,17 @@
+'use server';
+
+import { prisma } from '@repo/db';
+
+export async function searchUsers(query: string) {
+  return prisma.user.findMany({
+    where: {
+      name: {
+        contains: query,
```

Comment:
limit this to the max username length to prevent abuse.

Reply:
FIIIIINE